save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -56,18 +56,42 @@ class VulnerabilityController(
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched all vulnerabilities by filters")
     fun getAllVulnerabilities(
+        @RequestParam page: Int,
+        @RequestParam size: Int,
         @RequestBody filter: VulnerabilityFilter,
         authentication: Authentication?,
     ): Mono<VulnerabilityMetadataDtoList> = blockingToMono {
         if (
```

Comment:
may be extract this part into some method? its duplicated in 2 places

Reply:
Fixed